cmd/boltzcli/utils.go

```diff
@@ -114,3 +114,22 @@ func printDeposit(amount uint64, address string, hours float32, blockHeight uint
 	fmt.Printf("Please deposit %s into %s in the next ~%.1f hours (block height %d)\n",
 		amountString, address, hours, blockHeight)
 }
+
+func uintValidator(ans interface{}) error {
+	_, err := strconv.ParseUint(ans.(string), 10, 64)
```

Comment:
Are those casts safe to do?

Reply:
As long as you dont try to use them in a multi select (answer is a []string in that case) - yes